test/inferenceql/query/permissive_test.cljc

```diff
@@ -0,0 +1,16 @@
+(ns inferenceql.query.permissive-test
+  (:require [clojure.test :refer [are deftest]]
+            [inferenceql.query.parser :as parser]
+            [inferenceql.query.permissive :as permissive]))
+
+(deftest permissive-valid
+  (are [permissive strict] (= strict (-> permissive
+                                         (parser/parse :start :given-expr)
+                                         (permissive/->strict)
+                                         (parser/unparse)))
+    "model GIVEN VAR x = 0" "model CONDITIONED BY VAR x = 0"
```

Comment:
Is there any reason why this list of valid permissive expressions is shorter than the list in `parsert_test.cljc` (above, line 40 --49)?